### PR TITLE
Adding attendance from August 30th and 31st and hackerrank exercises from August 30th

### PR DESCRIPTION
**Pull request by ClaudiaHG23**

### CODE REVIEW

**Review by luisod78 — CHANGES_REQUESTED** [review]

File: ClaudiaHernandezGarcia/30Aug/HackerRank/arrayList/Solution.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Solution {
+    public static void main(String args[]) {
+        Scanner scan = new Scanner(System.in);
+        int lines = scan.nextInt();
+        List<ArrayList<Integer>> queryArray = new ArrayList<ArrayList<Integer>>();
```
Comment (luisod78) [inline ClaudiaHernandezGarcia/30Aug/HackerRank/arrayList/Solution.java:11]: I think you mean List<List<Integer>> Right ??

File: ClaudiaHernandezGarcia/30Aug/HackerRank/arrayList/Solution.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+        for (int index = 0; index < lines; index++) {
+
+            int numberOfIntegersInLine = scan.nextInt();
+            ArrayList<Integer> integersArray = new ArrayList<Integer>();
```
Comment (luisod78) [inline ClaudiaHernandezGarcia/30Aug/HackerRank/arrayList/Solution.java:16]: ups... same think here....
Code for interface not for implementation

**Review by luisod78 — CHANGES_REQUESTED** [review]

File: ClaudiaHernandezGarcia/30Aug/HackerRank/arrayList/Solution.java (hunk 1 above)
Comment (ClaudiaHG23) [reply ClaudiaHernandezGarcia/30Aug/HackerRank/arrayList/Solution.java]: I was wondering if both should have the interface. So should the list always be used, right?

**Review by luisod78 — APPROVED** [review]

**Review by edbermeo — APPROVED** [review]
LGTM!

**Pull request merged**